src/controllers/dynakube/dynakube_phase.go

```diff
@@ -0,0 +1,86 @@
+package dynakube
```

Comment:
wouldn't this fit better in `dynakube/status`?

Reply:
As there are only receiver functions for DynakubeController in there, I would say no

Reply:
`phase` is a property of `status` so this should be visible in the file structure as well